aspnet/signalr/overview/security/introduction-to-security.md

```diff
@@ -80,6 +80,13 @@ For each request, the server validates the contents of the token to ensure that
 
 Because the connection id is part of the verification process, you should not reveal one user's connection id to other users or store the value on the client, such as in a cookie.
 
+### Note: Connection Tokens vs Other Types of Tokens
```

Comment:
A couple of thoughts on the heading:
* Our style guide prescribes sentence casing for headings -- "... vs. other types of tokens"
* Consider making it H4 as it logically fits under the preceding Connection token heading.
* Consider dropping "Note: "